README.md

```diff
@@ -117,28 +142,9 @@ Additional resources:
 
 ### Run Time Dependencies
```

Comment:
I think We should remove all dependencies from here and from the pom,  but leave the pom be till some next pull request